package/contents/ui/lib/ConfigColor.qml

```diff
@@ -100,9 +100,9 @@ RowLayout {
 		visible: false
 		modality: Qt.WindowModal
 		title: configColor.label
-		showAlphaChannel: true
-		color: configColor.valueColor
-		onCurrentColorChanged: {
+		options: ColorDialog.ShowAlphaChannel
```

Comment:
Hmm, I don't remember this when creating `LibConfig.ColorField` so thanks I probably need to update the dialog code.

https://github.com/Zren/plasma-applet-lib/blob/master/package/contents/ui/libconfig/ColorField.qml#L160